src/platforms/dotnet/common/configuration/http-client-errors.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: HTTP Client Errors
+sidebar_order: 25
+description: "This feature, once enabled, automatically captures HTTP client errors, like bad response codes, as error events and reports them to Sentry"
+---
+
+Once enabled, this feature automatically captures HTTP client errors, like bad response codes, as error events and reports them to Sentry. The error event will contain the `request` and `response` data, such as `url`, `status_code`, and so on.
+
+To enable it:
+
+```csharp
+// Add this to the SDK initialization callback
+options.CaptureFailedRequests = true;
+```
+
+```fsharp
+// Add this to the SDK initialization callback
+options.CaptureFailedRequests <- true
+```
+
+HTTP client errors will then be captured for any requests from the Sentry SDK. To capture HTTP client errors for outbound requests from your own application, pass `SentryHttpMessageHandler` as the inner handler when creating your `HttpClient`:
+
+```csharp
+using var httpClient = new HttpClient(new SentryHttpMessageHandler());
+```
+
+```fsharp
+let httpClient = new HttpClient(new SentryHttpMessageHandler())
+```
+
+By default, only HTTP client errors with a response code between `500` and `599` are captured as error events, but you can change this behavior by setting the `FailedRequestStatusCodes` option:
+
+```csharp
+options.FailedRequestStatusCodes = new []{ new HttpStatusCodeRange(400, 599) };
```

Comment:
Though replacing the list is valid, I think it would less confusing in docs to show adding to it.
```csharp
options.FailedRequestStatusCodes.Add((400, 499));
```